fml/task_source.h

```diff
@@ -0,0 +1,61 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FML_TASK_SOURCE_H_
+#define FLUTTER_FML_TASK_SOURCE_H_
+
+#include "flutter/fml/delayed_task.h"
+#include "flutter/fml/task_queue_id.h"
+
+namespace fml {
+
+class MessageLoopTaskQueues;
+
+enum TaskSourceGrade {
+  kPrimary,
+  kSecondary,
+};
+
+class TaskSource {
```

Comment:
IMO, we should have a separate test for this in the FML test harness. Right now, the embedder unit-test is more of an integration test that might make it hard to test some of the subtleties of this interface.

Reply:
Done!